plugins/manage_engine_service_desk/icon_manage_engine_service_desk/actions/add_request_note/action.py

```diff
@@ -0,0 +1,35 @@
+import insightconnect_plugin_runtime
+
+from icon_manage_engine_service_desk.util.constants import Response, Note, ResponseStatus
+from .schema import AddRequestNoteInput, AddRequestNoteOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+class AddRequestNote(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="add_request_note",
+            description=Component.DESCRIPTION,
+            input=AddRequestNoteInput(),
+            output=AddRequestNoteOutput(),
+        )
+
+    def run(self, params: dict = None) -> dict:
+        request_id = params.get(Input.REQUEST_ID)
+        response_json = self.connection.api_client.add_request_note(
+            request_id=request_id,
+            description=params.get(Input.DESCRIPTION),
```

Comment:
Could you check if the description is a valid html string? What happens if it's not? Do we get an explicit error from manage engine?

Reply:
Manage Engine is not returning an error if it's not a valid HTML, also user can provide pure string.